modules/smaatoBidAdapter.js

```diff
@@ -342,6 +426,55 @@ function createVideoImp(bidRequest, videoMediaType) {
   };
 }
 
+function createNativeImp(bidRequest, nativeMediaType) {
+  const mainImageSize = nativeMediaType.image.sizes
```

Comment:
The documentation for it is still in progress, so the timing of this PR is a bit unfortunate, but prebid now supports native ORTB out of the box with very little adapter code required:

- `bidRequest.nativeOrtbRequest` should be an object very similar to your `createNativeImp(..).native`
- bids returned by `interpretResponse` can use the ORTB native response directly (`{native: {ortb: JSON.parse(bid.adm)}}`)

@musikele FYI

Reply:
thank you, i'm having a look and will update the PR